Fail compiling Quick using Carthage because of SwiftLint

Hi.
I try to compile and import Quick(v1.1.0) using Carthage(0.18.1), 
I encounter following error and fail to compile Quick.

```
carthage update --platform Mac --no-use-binaries
*** Fetching SnapKit
*** Fetching Nimble
*** Fetching Quick
*** Checking out Nimble at "v6.0.1"
*** Checking out Quick at "v1.1.0"
*** Checking out SnapKit at "3.2.0"
*** xcodebuild output can be found in /var/folders/3h/nxrqlf914yl8hyr2s5pmpv_80000gn/T/carthage-xcodebuild.U3vkZ0.log
*** Building scheme "Nimble-macOS" in Nimble.xcodeproj
*** Building scheme "Quick-OSX" in Quick.xcworkspace
** BUILD FAILED **


The following build commands failed:
	PhaseScriptExecution Run\ Script /Users/tabatakohei/Library/Developer/Xcode/DerivedData/Quick-bgdpjggnjjgrjqfwppjoscrzhjrp/Build/Intermediates/Quick.build/Release/Quick-macOS.build/Script-A870E6151DE00E37006891AD.sh
(1 failure)
~/MyMacProject/Carthage/Checkouts/Nimble/Sources/Lib/CwlPreconditionTesting/CwlPreconditionTesting/CwlCatchBadInstruction.swift:74:32: warning: Redundant Optional Initialization Violation: Initializing an optional variable with nil is redundant. (redundant_optional_initialization)
~/MyMacProject/Carthage/Checkouts/Nimble/Sources/Lib/CwlPreconditionTesting/CwlPreconditionTesting/CwlCatchBadInstructionPOSIX.swift:43:111: warning: Redundant Void Return Violation: Returning Void in a function declaration is redundant. (redundant_void_return)
~/MyMacProject/Carthage/Checkouts/Nimble/Sources/Nimble/DSL+Wait.swift:18:58: warning: Redundant Void Return Violation: Returning Void in a function declaration is redundant. (redundant_void_return)
~/MyMacProject/Carthage/Checkouts/Nimble/Sources/Nimble/DSL+Wait.swift:29:65: warning: Redundant Void Return Violation: Returning Void in a function declaration is redundant. (redundant_void_return)
~/MyMacProject/Carthage/Checkouts/Nimble/Sources/Nimble/DSL+Wait.swift:74:118: warning: Redundant Void Return Violation: Returning Void in a function declaration is redundant. (redundant_void_return)
~/MyMacProject/Carthage/Checkouts/Nimble/Sources/Nimble/DSL+Wait.swift:79:118: warning: Redundant Void Return Violation: Returning Void in a function declaration is redundant. (redundant_void_return)
~/MyMacProject/Carthage/Checkouts/Nimble/Sources/Nimble/DSL+Wait.swift:96:145: warning: Redundant Void Return Violation: Returning Void in a function declaration is redundant. (redundant_void_return)
~/MyMacProject/Carthage/Checkouts/Nimble/Sources/Nimble/FailureMessage.swift:15:40: warning: Redundant Optional Initialization Violation: Initializing an optional variable with nil is redundant. (redundant_optional_initialization)
~/MyMacProject/Carthage/Checkouts/Nimble/Sources/Nimble/FailureMessage.swift:16:40: warning: Redundant Optional Initialization Violation: Initializing an optional variable with nil is redundant. (redundant_optional_initialization)
~/MyMacProject/Carthage/Checkouts/Nimble/Sources/Nimble/Utils/Async.swift:30:44: warning: Redundant Optional Initialization Violation: Initializing an optional variable with nil is redundant. (redundant_optional_initialization)
~/MyMacProject/Carthage/Checkouts/Nimble/Tests/NimbleTests/Matchers/BeIdenticalToTest.swift:54:44: warning: Compiler Protocol Init Violation: The initializers declared in compiler protocols such as `ExpressibleByArrayLiteral` shouldn't be called directly. (compiler_protocol_init)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/Quick/NSString+C99ExtendedIdentifier.swift:15:21: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickFocusedTests/FocusedTests.swift:27:12: warning: Todo Violation: TODOs should be avoided: Port fitBehavesLike to Swift... (todo)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickFocusedTests/FocusedTests.swift:46:87: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickFocusedTests/main.swift:6:19: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTestHelpers/TestRun.swift:1: warning: Leading Whitespace Violation: File shouldn't start with whitespace: currently starts with 1 whitespace characters (leading_whitespace)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/Fixtures/FunctionalTests_SharedExamplesTests_SharedExamples.swift:15:85: error: Force Cast Violation: Force casts should be avoided. (force_cast)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/Fixtures/FunctionalTests_SharedExamplesTests_SharedExamples.swift:20: warning: Trailing Newline Violation: Files should have a single trailing newline. (trailing_newline)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/AfterEachTests.swift:25: warning: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/AfterEachTests.swift:31: warning: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/AfterEachTests.swift:37: warning: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/AfterEachTests.swift:41: warning: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/AfterEachTests.swift:51: warning: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/AfterEachTests.swift:47:97: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/AfterEachTests.swift:74:99: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/AfterEachTests.swift:90:89: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/AfterSuiteTests.swift:29:107: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/BeforeEachTests.swift:21: warning: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/BeforeEachTests.swift:25: warning: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/BeforeEachTests.swift:28: warning: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/BeforeEachTests.swift:33: warning: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/BeforeEachTests.swift:36: warning: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/BeforeEachTests.swift:59:101: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/BeforeEachTests.swift:75:92: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/BeforeSuiteTests.swift:29:103: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/Configuration/AfterEach/Configuration+AfterEachTests.swift:22:133: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/Configuration/BeforeEach/Configuration+BeforeEachTests.swift:19:133: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/DescribeTests.swift:10:103: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/ItTests.swift:26: warning: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/ItTests.swift:33: warning: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/ItTests.swift:40: warning: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/ItTests.swift:44: warning: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/ItTests.swift:51: warning: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/ItTests.swift:58: warning: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/ItTests.swift:117:71: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/ItTests.swift:24:53: warning: Opening Brace Spacing Violation: Opening braces should be preceded by a single space and on the same line as the declaration. (opening_brace)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/PendingTests.swift:35:135: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/SharedExamples+BeforeEachTests.swift:34:149: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/FunctionalTests/SharedExamplesTests.swift:43:119: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/main.swift:29:44: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
~/MyMacProject/Carthage/Checkouts/Quick/Sources/QuickTests/main.swift:35:50: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
~/MyMacProject/Carthage/Checkouts/Quick/Tests/QuickTests/QuickTests/FunctionalTests/AfterSuiteTests.swift:26:107: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)
A shell task (/usr/bin/xcrun xcodebuild -workspace ~/MyMacProject/Carthage/Checkouts/Quick/Quick.xcworkspace -scheme Quick-OSX -configuration Release ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES clean build) failed with exit code 65:
** BUILD FAILED **


The following build commands failed:
	PhaseScriptExecution Run\ Script /Users/tabatakohei/Library/Developer/Xcode/DerivedData/Quick-bgdpjggnjjgrjqfwppjoscrzhjrp/Build/Intermediates/Quick.build/Release/Quick-macOS.build/Script-A870E6151DE00E37006891AD.sh
(1 failure)
```

I think that SwiftLint should not be run when user try to compile and import Quick to their project,
so I'd like to suggest to add Aggregate to run SwiftLint script on only Test or Build.

realm-cocoa execute SwiftLint in that way.
https://github.com/realm/realm-cocoa

If it is okay, I'd like to send pr for this issue.

--- a/Quick.xcodeproj/project.pbxproj
+++ b/Quick.xcodeproj/project.pbxproj
@@ -6,6 +6,20 @@
 	objectVersion = 46;
 	objects = {
 
+/* Begin PBXAggregateTarget section */
+		732D8D691E516780008558BD /* SwiftLint */ = {
+			isa = PBXAggregateTarget;
+			buildConfigurationList = 732D8D6A1E516780008558BD /* Build configuration list for PBXAggregateTarget "SwiftLint" */;
+			buildPhases = (
+				732D8D6D1E516789008558BD /* ShellScript */,
+			);
+			dependencies = (
+			);
+			name = SwiftLint;
+			productName = SwiftLint;
+		};
+/* End PBXAggregateTarget section */
+
 /* Begin PBXBuildFile section */
 		1F118CDF1BDCA4AB005013A2 /* Quick.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 1F118CD51BDCA4AB005013A2 /* Quick.framework */; };
 		1F118CF51BDCA4BB005013A2 /* Quick.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 1F118CD51BDCA4AB005013A2 /* Quick.framework */; };
@@ -1223,7 +1237,7 @@
 			isa = PBXProject;
 			attributes = {
 				LastSwiftUpdateCheck = 0810;
-				LastUpgradeCheck = 0810;
+				LastUpgradeCheck = 0820;
 				ORGANIZATIONNAME = "Brian Ivan Gesiak";
 				TargetAttributes = {
 					1F118CD41BDCA4AB005013A2 = {
@@ -1246,6 +1260,10 @@
 						CreatedOnToolsVersion = 6.0;
 						LastSwiftMigration = 0800;
 						TestTargetID = 5A5D117B19473F2100F6D13D;
+					};
+					732D8D691E516780008558BD = {
+						CreatedOnToolsVersion = 8.2.1;
+						ProvisioningStyle = Automatic;
 					};
 					DA5663E71A4C8D8500193C88 = {
 						CreatedOnToolsVersion = 6.2;
@@ -1290,6 +1308,7 @@
 				1F118CDD1BDCA4AB005013A2 /* Quick - tvOSTests */,
 				1F118CEF1BDCA4BB005013A2 /* QuickFocused - tvOSTests */,
 				64076D0A1D6D7CEA00E2B499 /* QuickAfterSuite - tvOSTests */,
+				732D8D691E516780008558BD /* SwiftLint */,
 			);
 		};
 /* End PBXProject section */
@@ -1380,6 +1399,22 @@
 			runOnlyForDeploymentPostprocessing = 0;
 		};
 /* End PBXResourcesBuildPhase section */
+
+/* Begin PBXShellScriptBuildPhase section */
+		732D8D6D1E516789008558BD /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "if which swiftlint >/dev/null; then\n    swiftlint\nelse\n    echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi\n";
+		};
+/* End PBXShellScriptBuildPhase section */
 
 /* Begin PBXSourcesBuildPhase section */
 		1F118CD01BDCA4AB005013A2 /* Sources */ = {
@@ -2192,6 +2227,20 @@
 			};
 			name = Release;
 		};
+		732D8D6B1E516780008558BD /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				PRODUCT_NAME = "$(TARGET_NAME)";
+			};
+			name = Debug;
+		};
+		732D8D6C1E516780008558BD /* Release */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				PRODUCT_NAME = "$(TARGET_NAME)";
+			};
+			name = Release;
+		};
 		DA5663F11A4C8D8500193C88 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
@@ -2552,6 +2601,14 @@
 			);
 			defaultConfigurationIsVisible = 0;
 			defaultConfigurationName = Release;
+		};
+		732D8D6A1E516780008558BD /* Build configuration list for PBXAggregateTarget "SwiftLint" */ = {
+			isa = XCConfigurationList;
+			buildConfigurations = (
+				732D8D6B1E516780008558BD /* Debug */,
+				732D8D6C1E516780008558BD /* Release */,
+			);
+			defaultConfigurationIsVisible = 0;
 		};
 		DA5663F31A4C8D8500193C88 /* Build configuration list for PBXNativeTarget "QuickFocused - macOSTests" */ = {
 			isa = XCConfigurationList;

--- a/Quick.xcodeproj/project.pbxproj
+++ b/Quick.xcodeproj/project.pbxproj
@@ -986,7 +986,6 @@
 				1F118CD11BDCA4AB005013A2 /* Frameworks */,
 				1F118CD21BDCA4AB005013A2 /* Headers */,
 				1F118CD31BDCA4AB005013A2 /* Resources */,
-				A870E6171DE00FC7006891AD /* ShellScript */,
 			);
 			buildRules = (
 			);
@@ -1041,7 +1040,6 @@
 				5A5D117819473F2100F6D13D /* Frameworks */,
 				5A5D117919473F2100F6D13D /* Headers */,
 				5A5D117A19473F2100F6D13D /* Resources */,
-				A870E6161DE00E50006891AD /* ShellScript */,
 			);
 			buildRules = (
 			);
@@ -1178,7 +1176,6 @@
 				DAEB6B8A1943873100289F44 /* Frameworks */,
 				DAEB6B8B1943873100289F44 /* Headers */,
 				DAEB6B8C1943873100289F44 /* Resources */,
-				A870E6151DE00E37006891AD /* ShellScript */,
 			);
 			buildRules = (
 			);
@@ -1383,48 +1380,6 @@
 			runOnlyForDeploymentPostprocessing = 0;
 		};
 /* End PBXResourcesBuildPhase section */
-
-/* Begin PBXShellScriptBuildPhase section */
-		A870E6151DE00E37006891AD /* ShellScript */ = {
-			isa = PBXShellScriptBuildPhase;
-			buildActionMask = 2147483647;
-			files = (
-			);
-			inputPaths = (
-			);
-			outputPaths = (
-			);
-			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "if which swiftlint >/dev/null; then\nswiftlint; fi";
-		};
-		A870E6161DE00E50006891AD /* ShellScript */ = {
-			isa = PBXShellScriptBuildPhase;
-			buildActionMask = 2147483647;
-			files = (
-			);
-			inputPaths = (
-			);
-			outputPaths = (
-			);
-			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "if which swiftlint >/dev/null; then\nswiftlint; fi";
-		};
-		A870E6171DE00FC7006891AD /* ShellScript */ = {
-			isa = PBXShellScriptBuildPhase;
-			buildActionMask = 2147483647;
-			files = (
-			);
-			inputPaths = (
-			);
-			outputPaths = (
-			);
-			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "if which swiftlint >/dev/null; then\nswiftlint; fi";
-		};
-/* End PBXShellScriptBuildPhase section */
 
 /* Begin PBXSourcesBuildPhase section */
 		1F118CD01BDCA4AB005013A2 /* Sources */ = {

--- a/Quick.xcodeproj/xcshareddata/xcschemes/Quick-iOS.xcscheme
+++ b/Quick.xcodeproj/xcshareddata/xcschemes/Quick-iOS.xcscheme
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0810"
+   LastUpgradeVersion = "0820"
    version = "1.3">
    <BuildAction
       parallelizeBuildables = "YES"
@@ -17,6 +17,20 @@
                BlueprintIdentifier = "5A5D117B19473F2100F6D13D"
                BuildableName = "Quick.framework"
                BlueprintName = "Quick-iOS"
+               ReferencedContainer = "container:Quick.xcodeproj">
+            </BuildableReference>
+         </BuildActionEntry>
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "NO"
+            buildForProfiling = "NO"
+            buildForArchiving = "NO"
+            buildForAnalyzing = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "732D8D691E516780008558BD"
+               BuildableName = "SwiftLint"
+               BlueprintName = "SwiftLint"
                ReferencedContainer = "container:Quick.xcodeproj">
             </BuildableReference>
          </BuildActionEntry>

--- a/Quick.xcodeproj/xcshareddata/xcschemes/Quick-macOS.xcscheme
+++ b/Quick.xcodeproj/xcshareddata/xcschemes/Quick-macOS.xcscheme
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0810"
+   LastUpgradeVersion = "0820"
    version = "1.3">
    <BuildAction
       parallelizeBuildables = "YES"
@@ -17,6 +17,20 @@
                BlueprintIdentifier = "DAEB6B8D1943873100289F44"
                BuildableName = "Quick.framework"
                BlueprintName = "Quick-macOS"
+               ReferencedContainer = "container:Quick.xcodeproj">
+            </BuildableReference>
+         </BuildActionEntry>
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "NO"
+            buildForProfiling = "NO"
+            buildForArchiving = "NO"
+            buildForAnalyzing = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "732D8D691E516780008558BD"
+               BuildableName = "SwiftLint"
+               BlueprintName = "SwiftLint"
                ReferencedContainer = "container:Quick.xcodeproj">
             </BuildableReference>
          </BuildActionEntry>

--- a/Quick.xcodeproj/xcshareddata/xcschemes/Quick-tvOS.xcscheme
+++ b/Quick.xcodeproj/xcshareddata/xcschemes/Quick-tvOS.xcscheme
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0810"
+   LastUpgradeVersion = "0820"
    version = "1.3">
    <BuildAction
       parallelizeBuildables = "YES"
@@ -17,6 +17,20 @@
                BlueprintIdentifier = "1F118CD41BDCA4AB005013A2"
                BuildableName = "Quick.framework"
                BlueprintName = "Quick-tvOS"
+               ReferencedContainer = "container:Quick.xcodeproj">
+            </BuildableReference>
+         </BuildActionEntry>
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "NO"
+            buildForProfiling = "NO"
+            buildForArchiving = "NO"
+            buildForAnalyzing = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "732D8D691E516780008558BD"
+               BuildableName = "SwiftLint"
+               BlueprintName = "SwiftLint"
                ReferencedContainer = "container:Quick.xcodeproj">
             </BuildableReference>
          </BuildActionEntry>

